foundation-extensions/adobe-experience-platform-assurance/assurance-api-reference.md

```diff
@@ -6,79 +6,83 @@ Returns the current version of the AEP Assurance extension
 
 {% tabs %}
 {% tab title="Android" %}
-### Java
 
-### Syntax
+#### Java
```

Comment:
The ones on this page can use h3, otherwise it will show as bold Syntax/example below.